ArduPlane/Parameters.h

```diff
@@ -56,7 +56,7 @@ class Parameters {
         //
         k_param_auto_trim      = 10, // unused
         k_param_log_bitmask_old,  // unused
-        k_param_pitch_trim_cd,
+        k_param_pitch_trim_cd_old,  // replaced by pitch_trim
```

Comment:
we should keep the same ID

Reply:
I'm good with that - I didn't know it would work.

Reply:
fixed